### PR TITLE
EDU-1269 Add header strip to user pages

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1269

### CODE REVIEW

File: src/components/pages/admin.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ function Admin({ PageTemplate }) {
   return (
     <PageTemplate>
       <div className="AdminPage">
-        <h1>{t('pageNames:admin')}</h1>
+        <h1 className="u-page-title">{t('pageNames:admin')}</h1>
```
Comment (ahelmberger) [inline src/components/pages/admin.js:97]: We also have a new page merged in the meantime with a title: `comparison` :grinning:

File: src/components/default-page-template.less [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
   max-width: 100vw;
 }
 
 .DefaultPageTemplate-contentArea {
+  position: relative;
   flex: 1 0 0%;
   display: flex;
   align-items: flex-start;
-  position: relative;
+  flex-direction: column;
```
Comment (ahelmberger) [inline src/components/default-page-template.less:17]: This change makes the content of "fullscreen" pages like login not vertically centered anymore, so I guess we'll have to set it back to `flex-direction: row;` in `&.DefaultPageTemplate-contentArea--fullScreen`

**Comment by ahelmberger** [comment]
Looks super fancy :heart_eyes:

File: src/components/pages/admin.js (hunk 1 above)
Comment (adrianaluchian) [reply src/components/pages/admin.js:97]: I actually made a mental note on that and the forgot, thanks for reminding me!
Updated.

File: src/components/default-page-template.less (hunk 2 above)
Comment (adrianaluchian) [reply src/components/default-page-template.less:17]: Good catch 😬 Corrected.

File: src/components/pages/admin.js (hunk 1 above)
Comment (ahelmberger) [reply src/components/pages/admin.js:97]: :+1:

**Pull request merged**